profile.md

```diff
@@ -707,21 +707,21 @@ Since the user is a member of multiple groups (VOs) and also has the `/dune/dune
    </td>
   </tr>
   <tr>
-   <td><code>scope=wlcg.groups:/dune storage.read:/home/joe storage.write:/home/joe storage.read:/dune</code>
+   <td><code>scope=wlcg.capabilityset:/dune storage.read:/home/joe storage.write:/home/joe storage.read:/dune</code>
```

Comment:
This example is correct but it hides the semantics we discussed at the latest meeting.  The same answer would be retrieved without including `storage.read:/dune` in the request, because the wlcg.capabilityset request means "give me everything you've got related to this group".

Also, I'm wondering if there ever will be any capability set given to individuals apart from a VO.  I expect the /dune set will include things that people get for themselves for dune, for example read+write on /dune/joe.